package.json

```diff
@@ -2,13 +2,14 @@
   "name": "unistyle",
   "version": "0.1.3",
   "description": "Write modular and scalable CSS using the next version of ECMAScript.",
-  "main": "src/index.js",
+  "main": "build.js",
   "bin": {
     "unistyle": "bin/unistyle"
   },
   "scripts": {
-    "lint": "xo",
+    "lint": "xo --ignore build.js || true",
```

Comment:
Agreed, it is still sometimes annoying when let's say you comment out an assertion like
```js
assert.equal(typeof css, 'string');
```
and then it gives you an error that the variable `css` is not used and you gotta go back and remove the `css` parameter just for this test.

But otherwise I completely agree!